File: modul 4/PRAK405-2410817320004-AmandhaCitraMustika.c
#include <stdio.h>
int main(){
int a,b;
int total=0;
scanf("%d %d",&a, &b);
for(int i=1; i<=a; i++){
    int hasilbaris=0;

    for(int j=i;j>=1; j--){
        hasilbaris+=j*b;
        printf("(%d*%d)",j,b);
        if(j>1){
            printf("+");
        } }
printf("= %d\n",hasilbaris);
total+=hasilbaris;}
printf("%d",total);
    return 0;
}

File: modul 4/PRAK405-2410817320004-AmandhaCitraMustika.py
a,b=map(int,input().split())
total=0
for i in range(1,a +1):
    hasilbaris=0

    for j in range(i,0, -1):
        hasilbaris+=j*b
        print(f"({j}*{b})",end="")
        if j>1 :
            print("+",end="")

    print(f"={hasilbaris}")
    total+=hasilbaris

print(total)